src/api-request.js

```diff
@@ -0,0 +1,34 @@
+export const apiRequest = {
```

Comment:
Putting api request methods in this object will make it easily exportable and callable in other files.